cmd/snap-update-ns/change.go

```diff
@@ -498,7 +498,15 @@ func neededChangesImpl(currentProfile, desiredProfile *osutil.MountProfile) []*C
 		if reuse[current[i].Dir] {
 			changes = append(changes, &Change{Action: Keep, Entry: current[i]})
 		} else {
-			changes = append(changes, &Change{Action: Unmount, Entry: current[i]})
+			var entry osutil.MountEntry = current[i]
```

Comment:
typically we just write this as `entry := current[i]`.

Also, any reason we're copying the whole MountEntry struct around every time instead of using pointers?

Reply:
Ah, that `var` is a leftover from earlier experiment.

As for copying, it's not a big deal in this program but since there are pointers inside the mount entry I wanted to avoid having two entries sharing options by accident. Since go doesn't have const / mutability as a part of the variable type declaration it is hard to say "you can use this object but don't change it", hence I'm copying it as the entry is now distinct from the original.